### ISSUE TITLE
Templates: Our Team template renders incorrectly on mobile devices

### ISSUE THREAD

**Issue by apeatling** [Automattic/wp-calypso#37055]
Our team template includes image and text blocks that alternate left and right, allowing you to accomplish something like this on desktop:

<img width="1084" alt="Screen Shot 2019-10-24 at 4 07 43 PM" src="https://user-images.githubusercontent.com/1464705/67531791-e2050900-f678-11e9-93cf-c1434624c616.png">

However, on a mobile viewport the images and text collapse to one column. This is fine, but the right aligned images come after the left aligned images making things very hard to follow:

<img width="184" alt="Screen Shot 2019-10-24 at 4 10 10 PM" src="https://user-images.githubusercontent.com/1464705/67531836-0365f500-f679-11e9-9efa-a89a296d13c9.png">

In the usage of our team template, the small viewport collapsing this way makes it almost useless. The image should always come before the text.

The CSS to make this work is:

```
.wp-block-media-text.has-media-on-the-right.is-stacked-on-mobile .wp-block-media-text__media {
 grid-row: 1;
}
	
.wp-block-media-text.has-media-on-the-right.is-stacked-on-mobile .wp-block-media-text__content {
grid-row: 2;
}
```

This is fragile though, and is subject to the core block changing. This likely needs to be a setting in the block, and not CSS in the theme.

Some prior discussion has happened here: https://github.com/Automattic/wp-calypso/issues/36876

/cc @iamtakashi

**Comment by iamtakashi** [comment]
Found a related issue in Gutenberg. https://github.com/WordPress/gutenberg/issues/14283

**Comment by annacmc** [comment]
Closing up as the business themes and 'Our team template' for the business themes have been retired, additionally, fixes at https://github.com/Automattic/themes/issues/1521 and https://github.com/WordPress/gutenberg/pull/24468 will likely have resolved this issue.
